README.md

```diff
@@ -12,27 +12,32 @@ A Python implementation of WebGPU - the next generation GPU API.
 
 ## Introduction
 
-In short, this is a Python lib wrapping [wgpu-native](https://github.com/gfx-rs/wgpu)
-and exposing it with a Pythonic API similar to the [WebGPU spec](https://gpuweb.github.io/gpuweb/).
+In short, this is a Python lib wrapping
+[wgpu-native](https://github.com/gfx-rs/wgpu) and exposing it with a Pythonic
+API similar to the [WebGPU spec](https://gpuweb.github.io/gpuweb/).
```

Comment:
Just posting a comment to see what happens...

Reply:
Resolving makes the button go green. Unresolving disables the merge button again.